### PR TITLE
SaaS to GA

### PR DESCRIPTION
**Pull request by bakatzir**

### CODE REVIEW

**Comment by bakatzir** [comment]
@daryakoval / @amshamah419 WDYT about the RN?
```
 - Notes for the line: "SaaS Security is now Generally available."
   - Line is not using one of our templates, consider changing it to fit our standard.
     For more information run: `demisto-sdk doc-review --templates` or view our documentation at: https://xsoar.pan.dev/docs/documentation/release-notes
 - No misspelled words found in Packs/PrismaSaasSecurity/ReleaseNotes/1_2_11.md
```

File: Packs/PrismaSaasSecurity/ReleaseNotes/1_2_12.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#### Integrations
+##### SaaS Security
+- SaaS Security is now generally available.
```
Comment (bakatzir) [inline Packs/PrismaSaasSecurity/ReleaseNotes/1_2_12.md:3]:
```suggestion
- SaaS Security is now generally available.
- Updated the Docker image to: *demisto/python3:3.10.4.29342*.
```

**Comment by bakatzir** [comment]
@ShahafBenYakir / @yaakovi please consider force-merging:
```
[ERROR]: Packs/PrismaSaasSecurity/Integrations/SaasSecurity/SaasSecurity.yml: [RM110] - The following commands appear in SaasSecurity.yml but not in the README file:
get-modified-remote-data
update-remote-system
```

Alternatively, I can wait for the next sdk release.

**Comment by ShahafBenYakir** [comment]
@bakatzir Is this urgent?

**Comment by bakatzir** [comment]
> @bakatzir Is this urgent?

no

**Comment by ShahafBenYakir** [comment]
Tomorrow we release new SDK version, let's wait till then

**Pull request merged**